image-create

```diff
@@ -161,15 +166,18 @@ class MachineBuilder:
 
         partial = os.path.join(data_dir, self.machine.image + ".partial")
 
-        # Copy image via convert, to make it sparse again
-        subprocess.check_call(["qemu-img", "convert", "-c", "-O", "qcow2", self.machine.image_file, partial])
+        if not self.iso:
+            # Copy image via convert, to make it sparse again
+            subprocess.check_call(["qemu-img", "convert", "-c", "-O", "qcow2", self.machine.image_file, partial])
+        else:
+            subprocess.check_call(["cp", self.machine.image_file, partial])
```

Comment:
Copying is pointless and just a waste of time. That whole "rebuild" code path does not need to happen for iso files.

Reply:
Done.